include/picolibrary/spi.h

```
@@ -436,6 +436,134 @@ class GPIO_Output_Pin_Device_Selector : public GPIO_Output_Pin {
     }
 };
 
+/**
+ * \brief RAII SPI device selection guard.
+ *
```

Add note (`\attention`) explaining that deselection errors are ignored.